app/workers/delete_object_hierarchy_worker.rb

```diff
@@ -2,6 +2,12 @@
 
 # TODO: Rails 5 --> class DeleteObjectHierarchyWorker < ApplicationJob
 class DeleteObjectHierarchyWorker < ActiveJob::Base
+  include Sidekiq::Throttled::Worker
+
+  sidekiq_throttle({
```

Comment:
I am not sure if this should be done in `DeleteObjectHierarchyWorker` or in `DeletePlainObjectWorker`, or in both. Because `DeletePlainObjectWorker` is what actually hits the DB to destroy, but this one only does selects.
So I guess that this one should go as fast as it can and with as much parallelism as possible, and the other one is the one that we should actually limit a lot (but not just 10, maybe 50...).

But keep in mind that 10 at the same time in here is very little, that is even much less than one tenant opening all their assotiations at once (not deleting, because that is done by the other worker, but even only adding them in a worker for later).

Reply:
1. the thing is that the crash happened because of selecting, not because of deleting.. `DeletePlainObjectWorker` did great, it's just one query at the end.. but all loading it's happening in `DeleteObjectHierarchyWorker` which we need to control.. 

2. yeah, it's just for testing purposes.

Reply:
ok, good then 👍 💪